vllm/adapter_commons/request.py

```diff
@@ -1,8 +1,6 @@
 from abc import ABC, abstractmethod
-from dataclasses import dataclass
 
 
-@dataclass
```

Comment:
Why remove?

Reply:
It is not used anyway, and the children will use msgspec.Struct instead